core/src/main/java/org/apache/accumulo/core/metadata/schema/TabletMetadata.java

```diff
@@ -91,7 +94,8 @@ public class TabletMetadata {
   private boolean sawOldPrevEndRow = false;
   private Text endRow;
   private Location location;
-  private Map<StoredTabletFile,DataFileValue> files;
+  private List<Pair<String,String>> rawFiles;
```

Comment:
Would a multimap make more sense here?

Reply:
Since there were a few smaller changes that tried to reduce costs, I wonder which had the biggest impact, and which don't really matter. If there are a few that are more readable and don't really matter, that might be a good balance. Mostly, I'm worried about maintainability. However, if a follow on ticket is created to address this (by perhaps improving StoredTabletFile) in 3.x, we can probably defer any such readability improvements until then.